index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
I don't see 3rd assigment implemented:
Add a checkbox in front of the task and cross it out when the user checks it.